aten/src/ATen/gen.py

```diff
@@ -379,6 +378,21 @@ def filter_by_extension(files, *extensions):
     return filtered_files
 
 
+def cmpfiles_without_eol(a, b, names):
+    results = ([], [], [])    # match, mismatch, error
+    for x in names:
+        try:
+            ax = open(os.path.join(a, x), 'r').read().replace('\r', '').replace('\n', '')
+            bx = open(os.path.join(b, x), 'r').read().replace('\r', '').replace('\n', '')
```

Comment:
Wouldn't it be better to replace `\r\n` by `\n` and then `\r` by `\n` to have a normalization?
It's not that practically different, but I would be more accurate in expressing the intention.
Also, a comment why this is done ("normalize newlines for the benefit of Windows" or so) would help people to later understand what is going on and why.

Reply:
@t-vi thanks for comment.
your comment is right, I'll try to fix.